content/documentation/overview/index.adoc

```diff
@@ -115,6 +116,65 @@ Kiali will highlight configuration errors.
 image::istio-yaml-validation.png[Demo Website]
 {nbsp} +
 
+==== Validations performed
+[cols="2,1,2,1,1", options="header"]
+.List of destination rule validations
```

Comment:
is this the list of Kiali specific validations? the ones that are *not* done by Galley? or list of all validations together?

Reply:
they are the one not done by Galley, the specific ones created in Kiali.

Reply:
I would specify this in the documentation so it is clear these are "extra" ones that do not exist in Istio

Reply:
other than that, it is LGTM